docs/conf/portland/2019/cfp.rst

```diff
@@ -24,46 +23,78 @@ Between the talks on stage, the discussions in the unconference, the collaborati
 
 Our goal is to give documentarians a chance to connect and learn from each other. You'll have the chance to compare notes on what's happening in the industry, dig into questions of convention and good practice, and generally nerd out about all the things we love about documentation.
 
+This year, as usual, we're trying to iterate on our tried and tested processes, both to improve our program and to give you, the speakers, a better idea of what to expect.
+
 What we’re looking for
 ----------------------
 
-**Mix of topics and perspectives**
-
-The focus of Write the Docs is software documentation, but we actively seek talks that address a wide range of related subjects,
-at various levels of expertise. Documentation perspectives from other fields are welcome! Talks from the scientific community, fiction writers, system administrators, and support staff – in addition to technical writers and software developers – are all valuable to our attendees.
-
-**Diverse group of speakers**
+**Mix of topics**
 
-We welcome talks from first-time speakers, from industry experts, and from everyone in between. Whatever your background and experience, we prefer hearing about new approaches than about tried-and-tested technology. We especially welcome talks from underrepresented groups within the tech community.
+The focus of Write the Docs is software documentation, but we actively seek talks that address a wide range of related subjects, at various levels of expertise.
+Documentation perspectives from other fields are welcome, as are topics from adjacent fields!
```

Comment:
Maybe add a "who do we want to hear from" section, with a list of "including but not limited to" and then:
- Writers
- Developers
- Designers
- Testers
- Support
- Community
- Science
- Education
- Etc etc etc

?